public/index.html

```diff
@@ -26,7 +26,7 @@
     -->
     <title>React App</title>
   </head>
-  <body>
+  <body style="background-color: #2d4059">
```

Comment:
Has nothing to do with the HomeBlog, and changing an element style in the html elements besides by changing it in a css file is a bad practice